dts/bindings/cpu/cpu.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: CPU Base Structure
+version: 0.1
+
+description: >
+    This binding gives the base structures for all CPUs
+
+inherits:
+    !include base.yaml
+
+properties:
+    clock-frequency:
+      type: int
+      category: optional
+      description: Clock frequency in Hz
+      generation: define
+
+    clocks:
```

Comment:
Is there a case we have a clock property on a cpu node?

Reply:
No, I checked. Should I remove this for now?

Reply:
Checked with @pizi-nordic. According to the spec the frequency either comes from `clock-frequency` or from `clocks` so it is here for completeness with the spec.

Reply:
Ok, leave it in than.